packages/app-contracts/src/Deploy.tsx

```diff
@@ -256,8 +256,6 @@ class Deploy extends ContractModal<Props, State> {
     if (record) {
       const address = record.event.data[1] as any as AccountId;
 
-      await apiUi.isReady;
```

Comment:
No need, components are only rendered when API is ready

Reply:
Got it!